### DISCUSSION TITLE
how to get correct stats with returns freq different from daily

### DISCUSSION THREAD

**Discussion by Visema** [ranaroussi/quantstats#312] [General]
Code has clearly been written to deal with daily returns but often available historical data is only available with a weekly or monthly cadence. Feeding other than daily returns obviously causes quantstats to produce wrong stats. Q: Would it be possible to add a returns freq key so that all stats are adjusted accordingly and correctly calculated

**Comment by rosed777** [comment]
Yes - I'd like a monthly option. Is there a workaround that we can use for weekly or monthly?
